packages/web3/types/index.d.ts

```diff
@@ -0,0 +1,53 @@
+/*
+    This file is part of web3.js.
+    web3.js is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    web3.js is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+    You should have received a copy of the GNU Lesser General Public License
+    along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * @file index.d.ts
+ * @author Josh Stevens <joshstevens19@hotmail.co.uk>
+ * @date 2018
+ */
+
+import {AbstractWeb3Module, Web3ModuleOptions, Providers} from 'web3-core';
+import {Utils} from 'web3-utils';
+import * as net from 'net';
```

Comment:
I think ```import * as net from 'net'``` is not needed this is just for the nodejs runtime and you will have there the types.

Reply:
this is a types file, i am not sure that would work? I think you need to `import * as net from 'net'`. For example even though fs is built into node you still need to import it to use it. Think it has to stay as it is and i think should as its more explicit of what namespace its coming from.. thoughts?